examples/historic_async.py

```diff
@@ -0,0 +1,126 @@
+import sys
+import asyncio
+import alpaca_trade_api as tradeapi
+from alpaca_trade_api.rest import TimeFrame, URL
+from alpaca_trade_api import rest_async
+from alpaca_trade_api.rest_async import gather_with_concurrency
+from alpaca_trade_api import AsyncRest
+import pandas as pd
+
+NY = 'America/New_York'
+
+
+async def get_historic_bars(symbols, start, end, timeframe: TimeFrame):
+    major = sys.version_info.major
+    minor = sys.version_info.minor
+    if major < 3 or minor < 6:
+        raise Exception('asyncio is not support in your python version')
```

Comment:
In my opinion this check should go to the SDK.

Reply:
in order to do that I need to move `get_historic_bars(symbols, start, end, timeframe: TimeFrame)` inside the sdk. 
right now I place it in the example code and this is used to get data for multiple symbols. 

do you think getting data for multiple symbols should sit inside the sdk?

(btw, this issue was very hard to understand on why I'm getting different results on different python versions)

Reply:
I understand. For now it's okey this way. In this case the only thing I'd do is I refactored the common parts of `get_historic_bars`, `get_historic_trades` and `get_historic_quotes` to a common base function.

Reply:
done